TASK1.3/main.py

```diff
@@ -0,0 +1,17 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+
+# *Пример:*
+
+# 385916 -> yes
+# 123456 -> no
+
+x = [int(x) for x in list(input())]
```

Comment:
Переводите введенную ставку в виде числа в список необязательно.
```suggestion
x = [int(x) for x in input()]
```